### PR TITLE
Added a bias term in Bahdanau attention alignments

### PR DESCRIPTION
**Pull request by rubenvereecken**
The original paper by Bahdanau (https://arxiv.org/abs/1409.0473) lists a bias term (Eq. 18) and Tensorflow's normalized Bahdanau implementation uses one, so I don't see why we shouldn't use one in the original implementation.

Edit: Err, I was wrong, staring at papers too much all day, and quoted an equation from the paper that introduces Bahdanau energy normalization (http://arxiv.org/abs/1704.00784). Still, the original may have used it too as they state they "omit bias terms to make the equations less cluttered". My bad - the reviewer decides what to do with this I guess.

### CODE REVIEW

**Comment by jhseu** [comment]
@lmthang Mind taking a look?

**Comment by drpngx** [comment]
@lmthang any luck with this?

**Comment by sb2nov** [comment]
@lmthang ping

**Comment by lmthang** [comment]
Hi guys,

Sorry for a delay on this. 

There was no bias term in the original paper by Baudanau https://arxiv.org/pdf/1409.0473.pdf (see page 14). This change will break existing models that uses "bahdanau" attention, so I will oppose this change.
(The author of this PR also acknowledged that he/she misread the paper).

If people would like to have bias in the attention, I'd recommend using normed_bahdanau instead. See https://github.com/tensorflow/nmt#hyperparameters.

-Thang

**Comment by ebrevdo** [comment]
Closing...
